libs/wingsdk/src/std/json.ts

```diff
@@ -42,15 +49,18 @@ export class Json {
    * Formats Json as string
    *
    * (JSON.stringify($args$))
```

Comment:
I know it's not part of your change, but this macro-looking thing probably shouldn't be here

```suggestion
```